### DISCUSSION TITLE
Mosfet Threshold?

### DISCUSSION THREAD

**Discussion by letGitFoxy** [TeamOpenFIRE/OpenFIRE-Firmware#33] [Q&A] [answered]
Hi there. Im hoping to be able to swap the IRL540 for something smaller and surface mountable. Can anyone tell me the voltage that is being applied to the gate? 

Im hoping its possible to use a mosfet with a SOT-23 package.
Any other technical aspects I should be aware of are welcomed.
Thanks.

**Comment by SawdustMaker** [comment, marked as answer]
The rp2040 digital outputs are 3.3v.
